### PR TITLE
Add React Navbar 

### PR DESCRIPTION
**Pull request by Moejoe90**
navbar

### CODE REVIEW

File: app/components/CommitNavbar.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@import "../../common/sass/variables.scss";
+
+.brand {
+  height: 40px;
+  width: 110px;
+  background: url(../../page/assets/NeuFund_icon_white.png);
```
Comment (Moejoe90) [inline app/components/CommitNavbar.scss]: This will be changed to the real icon

File: test/components/CommitNavbar.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+describe("<CommitNavbar />", () => {
+  it("should render", () => {
+    const component = shallow(
+      <CommitNavbar>
+        <div className="child" />
+      </CommitNavbar>
+    );
+    expect(component).to.contain(CancelCommit);
+  });
+});
```
Comment (Moejoe90) [inline test/components/CommitNavbar.spec.tsx]: Not much to test

File: app/components/CommitNavbar.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  background-size: 30px;
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  padding: 0;
+  border-right: 1px solid #FFF;
```
Comment (krzkaczor) [inline app/components/CommitNavbar.scss]: use $white for color

File: app/components/CommitNavbar.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  background-repeat: no-repeat;
+  padding-top: 6px;
+  padding-bottom: 42px;
+  padding-right: 146px;
+  background-position: center;
+}
+
+.header {
+  background: linear-gradient(#2F576A 2.36%, #325566 100%);;
+  border-color: #FFF;
```
Comment (krzkaczor) [inline app/components/CommitNavbar.scss]: variable for color here

File: app/components/CommitNavbar.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.commit {
+   margin-left: 25px;
+   margin-top: 18px;
+   color: #FFF;
```
Comment (krzkaczor) [inline app/components/CommitNavbar.scss]: variable for color here

File: app/components/CommitNavbar.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { Nav, Navbar } from "react-bootstrap";
+import * as styles from "./CommitNavbar.scss";
+
+// it is currently assumed that Cancel component will change the state and clear everything
+export const CancelCommit: React.SFC = () => <a href="#" className={styles.clear} />;
+
+export const CommitNavbar: React.SFC = () =>
```
Comment (krzkaczor) [inline app/components/CommitNavbar.tsx]: This component should be able to receive and display children (title etc).  This should be tested as well
image

File: app/components/CommitNavbar.scss (hunk 3 above)
Comment (Moejoe90) [reply app/components/CommitNavbar.scss]: done

File: app/components/CommitNavbar.scss (hunk 4 above)
Comment (Moejoe90) [reply app/components/CommitNavbar.scss]: done

File: app/components/CommitNavbar.scss (hunk 5 above)
Comment (Moejoe90) [reply app/components/CommitNavbar.scss]: done

File: test/components/CommitNavbar.spec.tsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  it("should render", () => {
+    const message = "Hello you";
+    const component = shallow(
+      <CommitNavbar>
+        <div className="child">
+          {message}
+        </div>
+      </CommitNavbar>
+    );
+    expect(component.find(".child")).to.have.length(1);
```
Comment (krzkaczor) [inline test/components/CommitNavbar.spec.tsx]: 👍

File: app/components/CommitNavbar.scss [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  background-size: 30px;
+  background-repeat: no-repeat;
+  padding-top: 6px;
+  padding-bottom: 42px;
+  padding-right: 146px;
+  background-position: center;
+}
+
+.header {
+  background: linear-gradient(#2F576A 2.36%, #325566 100%);
```
Comment (MostafaBalata) [inline app/components/CommitNavbar.scss]: colors should be from the variables.scss file

File: yarn.lock [hunk 9]
```diff
@@ -451,14 +451,7 @@ babel-messages@^6.23.0:
...
 
-babel-runtime@^6.11.6, babel-runtime@^6.22.0:
-  version "6.25.0"
-  resolved "https://registry.yarnpkg.com/babel-runtime/-/babel-runtime-6.25.0.tgz#33b98eaa5d482bb01a8d1aa6b437ad2b01aec41c"
-  dependencies:
-    core-js "^2.4.0"
-    regenerator-runtime "^0.10.0"
-
-babel-runtime@^6.26.0:
+babel-runtime@^6.11.6, babel-runtime@^6.22.0, babel-runtime@^6.26.0:
```
Comment (krzkaczor) [inline yarn.lock]: why changing `yarn.lock`? no new deps were added

**Pull request merged**